common/scripted_triggers/wc_scripted_triggers.txt

```diff
@@ -1012,6 +1013,8 @@ is_medium_humanoid_trigger = {
 			trait = creature_frostnymph
 			trait = creature_highborne
 			trait = creature_sethrak
+			trait = creature_wickerman # sort of?
+			trait = creature_wicker_behemoth # sort of?
```

Comment:
Medium? Realy?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79052686-5e040500-7c49-11ea-8b2f-b41024781491.png)

</details>